### ISSUE TITLE
Support "Final" Stage

### ISSUE THREAD

**Issue by twotired** [mibexsoftware/bamboo-plan-dsl-plugin#96]
There seems to be no way to configure a Stage to be final. It should be possible to define a final stage that runs, even after a failed task.

If a stage is set to "FINAL" in the UI, and then I look at the configure -> DSL tab, there is no mention of the final setting.

**Comment by mrueegg** [comment]
You're right, support for final stages is not yet implemented. We will address this with the next release of the app. I'll reply back here once it is available. Thanks, Michael
